ex03.ex

```diff
@@ -149,7 +177,19 @@ defmodule Ex03 do
   Think a little about a nice way to lay this code out.
   """
 
-  def won . . . "your code"
+  #def won . . . "your code"
```

Comment:
## -3

The object of the assignment was to use pattern matching, and doing so leads to a really nice layout, such as:

~~~ elixir
  def won({ a, a, a,
            _, _, _,
            _, _, _ }), do: a

  def won({ _, _, _,
            a, a, a,
            _, _, _ }), do: a

  def won({ _, _, _,
            _, _, _,
            a, a, a }), do: a

  def won({ a, _, _,
            a, _, _,
            a, _, _ }), do: a

  def won({ _, a, _,
            _, a, _,
            _, a, _ }), do: a

  def won({ _, _, a,
            _, _, a,
            _, _, a }), do: a

  def won({ a, _, _,
            _, a, _,
            _, _, a }), do: a

  def won({ _, _, a,
            _, a, _,
            a, _, _ }), do: a

  def won(_), do: false
~~~

or

~~~ elixir
  def won1({ a, a, a,  _, _, _,  _, _, _ }), do: a
  def won1({ _, _, _,  a, a, a,  _, _, _ }), do: a
  def won1({ _, _, _,  _, _, _,  a, a, a }), do: a
  def won1({ a, _, _,  a, _, _,  a, _, _ }), do: a
  def won1({ _, a, _,  _, a, _,  _, a, _ }), do: a
  def won1({ _, _, a,  _, _, a,  _, _, a }), do: a
  def won1({ a, _, _,  _, a, _,  _, _, a }), do: a
  def won1({ _, _, a,  _, a, _,  a, _, _ }), do: a
  def won1(_), do: false
~~~